.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Deployment
+
+on: [release]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args:
+                      build -a -c --skip-tests
+            - name: Ballerina Push
```

Comment:
Add 
`env:
                  JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true` 
here too.

Reply:
Added it in L23